tripper/triplestore.py

```diff
@@ -539,6 +539,14 @@ def list_databases(cls, backend: str, **kwargs):
     # interfaces to the triples(), add_triples() and remove() methods
     # implemented by all backends.
 
+    prefer_sparql = property(
+        fget=lambda self: getattr(self.backend, "prefer_sparql", None),
+        doc=(
+            "Whether the backend prefer SPARQL over the triples() interface. "
+            "Is None if not specified by the backend."
```

Comment:
```suggestion
            "Whether the backend uses SPARQL and not the triples() interface. "
            "Is None if not specified by the backend."
```

Actually I do not entirely understand what this is. Can you exlpain it better so we can improve the docstring?

Reply:
This is really about preferences. There is a big difference between backends, some can be used very efficient with Triplestore.triples() (e.g. collection, ontopy, ...) while other are very efficient with Triplestore.query() (e.g. fuseki, graphdb, ...). 

Even though Tripper requires that the Triplestore.triples() is implemented, we have to use Triplestore.query() for some backends (like fuseki) because of the issue with blank nodes getting renamed. The purpose of this PR is to allow a user of tripper to automatically select the most appropriate interface depending on the current backend.

Reply:
This you can write in the docstring.

Reply:
done